models/cow_protocol/ethereum/cow_protocol_ethereum_sources.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+sources: 
+  - name: gnosis_protocol_v2_ethereum
+    description: "Ethereum decoded tables related to CoW Protocol contract"
+    tables:
+      - name: GPv2AllowListAuthentication_evt_SolverAdded
+        description: "GPv2AllowListAuthentication list of events when a Solver was added"
+        columns:
+          - &solver
+            name: solver
+            description: "Solver wallet address"
+          - &contract_address
+            name: contract_address
+            description: "Ethereum address for the liquidity pool used in transaction"
+          - &evt_block_number
+            name: evt_block_number
+            description: "Block number which processed the unique transaction hash"
+          - &evt_block_time
+            name: evt_block_time
+            description: "Timestamp for block event time in UTC"
+          - &evt_index
+            name: evt_index
+            description: "Index value of the transaction"
+          - &evt_tx_hash
+            name: evt_tx_hash
+            description: "Primary key of the transaction"
+            tests:
+              - not_null
```

Comment:
Does this mean that "tests do exist"? I would have expected this to be a reference to the test file or something.

Reply:
Here this means that a test has been set up to check if evt_tx_hash column values are not NULL. It'll fail if that's the case